docs/metrics_output.log

```diff
@@ -0,0 +1,898 @@
+# HELP ovms_requests_success Number of successful requests to a model or a DAG.
```

Comment:
it is not a log but output so maybe .out extension would be better
add this file to exception for license header check

Reply:
Ok.